src/components/LanguagePicker.tsx

```diff
@@ -46,14 +43,14 @@ export const LanguagePicker: React.FC = () => {
 
       return (
         <Pressable {...props}>
-          <HStack>
+          <Row>
             <Text fontSize="xl" pr="2">
               {languages[language].emoji}
             </Text>
             <Animated.View style={[animatedIconStyle, styles.icon]}>
-              <ChevronDownIcon />
+              <Icon size={24} name="arrow-down-line" />
```

Comment:
It makes the Icon invisible on a light theme

Reply:
Update:
Icon isn't white by default, without color it is transparent - so not visible at all

<img width="634" alt="image" src="https://github.com/binarapps/expo-ts-template/assets/24510246/11b1a291-39f1-434c-ad80-ebe58cf00171">

Reply:
I've updated icon (check pr description)